src/drawmembrane/draw_membrane2a.c

```diff
@@ -507,13 +532,26 @@ int main(int argc, char *argv[])
   float V=0, I=0.1, sdie, cdie, pdie, mdie, idie;
   float l_m=40, a0=0;
   float z_m0=0, R_m0=0, R_m1=0, x0_R=NO_COORDINATE, y0_R=NO_COORDINATE;
-  float R, R_temp;
+  float R, R2, R_temp;  /*11-2021 added R2*/
   char *infix;
   char *file_name_x, *file_name_y, *file_name_z;
   char *file_name_k, *file_name_c;
   char *f1, *f2, *f3, *f4, *f5, *f6;
   char ext[]="m.dx";
   bool compression = FALSE;
+
+  /*11-2021*/
```

Comment:
remove date from comment, that's available in the git log, rest of the comment is great